web-local/src/lib/components/leaderboard/LeaderboardHeader.svelte

```diff
@@ -1,23 +1,84 @@
-<script>
-  export let isActive = false;
+<script lang="ts">
+  import { EntityStatus } from "@rilldata/web-local/common/data-modeler-state-service/entity-state-service/EntityStateService";
+
+  import Spinner from "@rilldata/web-local/lib/components/Spinner.svelte";
+  import Shortcut from "@rilldata/web-local/lib/components/tooltip/Shortcut.svelte";
+  import Tooltip from "@rilldata/web-local/lib/components/tooltip/Tooltip.svelte";
+  import TooltipContent from "@rilldata/web-local/lib/components/tooltip/TooltipContent.svelte";
+  import TooltipShortcutContainer from "@rilldata/web-local/lib/components/tooltip/TooltipShortcutContainer.svelte";
+  import TooltipTitle from "@rilldata/web-local/lib/components/tooltip/TooltipTitle.svelte";
+  import LeaderboardOptionsMenu from "./LeaderboardOptionsMenu.svelte";
+
+  export let displayName: string;
+  export let isFetching: boolean;
+  export let toggleFilterExcludeMode: () => void;
```

Comment:
We should use dispatch instead of passing callbacks. Please change in other components as well.